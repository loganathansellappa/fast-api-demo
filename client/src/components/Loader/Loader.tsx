export const Loader = () => {
  return (
    <div className="flex flex-col items-center justify-center h-screen">
      <h1 className="text-2xl font-bold mb-4">Request is processing!</h1>
    </div>
  );
};
