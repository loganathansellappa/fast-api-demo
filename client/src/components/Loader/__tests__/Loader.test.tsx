import { render } from "@testing-library/react";
import { Loader } from '../Loader.tsx';

let element: { container: HTMLElement };

test("renders Loader component", () => {
  element = render(<Loader />);
  expect(element.container.firstChild).toMatchSnapshot();
  expect(element.container.textContent).toContain("Request is processing!");
});