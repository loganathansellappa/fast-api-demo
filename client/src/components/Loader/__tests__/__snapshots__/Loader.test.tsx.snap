// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loader component 1`] = `
<div
  class="flex flex-col items-center justify-center h-screen"
>
  <h1
    class="text-2xl font-bold mb-4"
  >
    Request is processing!
  </h1>
</div>
`;
